wodles/aws/aws_s3.py

```diff
@@ -2765,19 +2787,28 @@ def get_alerts(self):
             last_scan = initial_date
 
         datetime_last_scan = datetime.strptime(last_scan, '%Y-%m-%d %H:%M:%S.%f')
+        date_scan = datetime_last_scan if not self.only_logs_after or datetime_last_scan > \
+            (date_ol := datetime.strptime(self.only_logs_after, "%Y%m%d")) else date_ol
```

Comment:
```suggestion
        date_scan = datetime_last_scan if not self.only_logs_after or datetime_last_scan > (
            datetime_only_logs := datetime.strptime(self.only_logs_after, "%Y%m%d")) else datetime_only_logs
```
Use a better naming convention for the `date_ol` variable.

Reply:
Fixed in 7fa3954!